client/src/components/game/eventLog/events/PlayerCombatStarEvent.vue

```diff
@@ -149,26 +149,43 @@ export default {
     },
     getStarShape () {
       return GameHelper.getPlayerById(this.$store.state.game, this.event.data.playerIdOwner).shape;
+    },
+    totalDefenderShips(index) {
```

Comment:
`totalDefenderShips` and `totalAttackerShips` can be merged into one.

Maybe something like `totalShips(star, carriers, index)` and pass in `null` to the `star` parameter for the attacker.